osf/external/gravy_valet/request_helpers.py

```diff
@@ -56,6 +57,12 @@ def get_account(gv_account_pk, requesting_user):  # -> JSONAPIResultEntry
         params={'include': ACCOUNT_EXTERNAL_STORAGE_SERVICE_PATH},
     )
 
+def get_links(node_guid, requesting_user=None):
+    return iterate_gv_results(
+        ADDONS_ENDPOINT.format(addon_type=AddonType.LINK) + f'/{node_guid}/verified-links',
+        requesting_user=requesting_user,
+    )
```

Comment:
I think we already have this implemented (which was newly merged yesterday morning, see https://github.com/CenterForOpenScience/osf.io/pull/11113/files#diff-8d04c47c31d3869aca32c85256dc4dc08e69fd836971299d95b8386503348668), please take a look and I think the new `get_verified_links()` should provide what you need.